pkg/apis/eksctl.io/v1alpha5/cloudwatch.go

```diff
@@ -1,16 +1,17 @@
 package v1alpha5
 
-// ClusterCloudWatch contains config parameters related to CloudWatch
-type ClusterCloudWatch struct {
-	//+optional
-	ClusterLogging *ClusterCloudWatchLogging `json:"clusterLogging,omitempty"`
-}
-
-// ClusterCloudWatchLogging container config parameters related to cluster logging
-type ClusterCloudWatchLogging struct {
-	//+optional
-	EnableTypes []string `json:"enableTypes,omitempty"`
-}
+type (
+	// ClusterCloudWatch contains config parameters related to CloudWatch
```

Comment:
because of limitations of the go internal `ast` machinery, it appears to be much harder to get access to the comments on a struct if that struct is declared top level like `type X struct {...}`. If all structs are inside a `type ( X struct {...} )` the code can stay simpler.

Reply:
Pasting some more info here:

> the type X struct variant is parsed differently (as a GenDecl). and at the point where I look in the package scope and find the TypeSpec for X, those comments are missing because they're on the GenDecl which contains that TypeSpec. with type ( X struct ) the comments are on the TypeSpec itself.
> i think I'd have to go back over the files belonging to the package, check all GenDecls, look into the Scope of the package and replace the comments on TypeSpecs with the comments on the GenDecls.

Reply:
hmm I fixed it, it's actually not too bad